src/prefect/cli/agent.py

```diff
@@ -43,27 +43,31 @@ def agent():
 
 
 @agent.command(hidden=True)
-@click.argument("name", default="local")
+@click.argument("agent-option", default="local")
 @click.option(
     "--token", "-t", required=False, help="A Prefect Cloud API token.", hidden=True
 )
 @click.option(
     "--verbose", "-v", is_flag=True, help="Enable verbose agent logs.", hidden=True
 )
+@click.option(
+    "--name", "-n", required=False, help="A name to use for the agent", hidden=True
```

Comment:
could you explicitly set `default=None` here? I'm worried about a future in which click passes `""` as the name and it gets used accidentally.